quicksilver/assetlist.json

```diff
@@ -473,6 +473,264 @@
           }
         }
       ]
+    },
+    {
+      "description": "Quicksilver Liquid Staked ARCH",
+      "denom_units": [
+        {
+          "denom": "aqarch",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "qarch",
+          "exponent": 18,
+          "aliases": []
+        }
+      ],
+      "base": "aqarch",
+      "name": "Quicksilver Liquid Staked ARCH",
+      "display": "qarch",
+      "symbol": "qARCH",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "archway",
+            "base_denom": "aarch"
+          },
+          "provider": "Quicksilver"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qarch.png",
```

Comment:
Note that it is strongly recommended (and required for sever cases) to remove all margin/padding from token images. Many of the uploaded images have transparent padding to varying degrees.

Reply:
Done.